crates/governance/src/vp/mod.rs

```diff
@@ -28,7 +28,7 @@ use crate::ProposalVote;
 pub const ADDRESS: Address = Address::Internal(InternalAddress::Governance);
 
 /// The maximum number of item in a pgf proposal
-pub const MAX_PGF_ACTIONS: usize = 20;
+pub const MAX_PGF_ACTIONS: usize = 1000;
```

Comment:
could this be even higher for more flexibility or are there any concerns?

Reply:
I'm not sure if there are concerns over it being higher, @Fraccaman? If not, I'm not opposed to making it higher

Reply:
no idea, we didn't do any benchmarks. Im just a little bit concerned about the possibility of a pgf proposal with 1000 ibc transfers

Reply:
The particular immediate use case (donor drop) are all native transfers FWIW, but I see your point.

What very specifically are your concerns? Let's test on a devnet or smth